codespell_lib/data/dictionary.txt

```diff
@@ -24991,6 +24991,7 @@ srinked->shrunk
 srinking->shrinking
 srollbar->scrollbar
 srouce->source
+srting->string
```

Comment:
Can you also suggest sorting.

Can you add the plural version of this typo too.

Reply:
Oh right, it didn't occur to me. Will do both changes shortly.

Reply:
@peternewman done :)